lgc/patch/PatchWaveSizeAdjust.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ ***********************************************************************************************************************
+ *
+ *  Trade secret of Advanced Micro Devices, Inc.
+ *  Copyright (c) 2021, Advanced Micro Devices, Inc., (unpublished)
+ *
+ *  All rights reserved. This notice is intended as a precaution against inadvertent publication and does not imply
+ *  publication or any waiver of confidentiality. The year included in the foregoing notice is the year of creation of
+ *  the work.
+ *
+ **********************************************************************************************************************/
+/**
```

Comment:
@xuechen417 this is closed source license, could you change it to open source license?

Reply:
Thanks. Done.